CONTRIBUTING.md

```diff
@@ -38,11 +38,47 @@ We are also using
 - [Vite](https://github.com/vuejs/vite) - a speedy Vue.js framework
 - [Monaco](https://microsoft.github.io/monaco-editor/) - the editor from Visual Studio Code
 - [Bulma](https://bulma.io/) - a CSS framework
-- [Ant Design Vue](https://2x.antdv.com/components/overview/) - a components framework (still testing it out)
+- [Ant Design Vue](https://2x.antdv.com/components/overview/) - a components framework (still testing it out, use as sparingly as possible right now)
 
 ### Code Structure
 
-- `/src/pages/` contains ...
-  TODO: Write more
+- `/src/pages/` Contains all registered (accessible via its own link) pages. See the chapter Developing a new page for more informations
```

Comment:
I think links to a different Markdown header work on GitHub. `[Registering a new page](#registering-a-new-page)`

Reply:
added the links